coalib/bearlib/languages/__init__.py

```diff
@@ -16,8 +17,10 @@
 from .definitions.html import HTML
 from .definitions.Java import Java
 from .definitions.JavaScript import JavaScript
+from .definitions.Jinja2 import Jinja2
```

Comment:
please do not re-arrange these.  That isnt what your issue is about.

Create a newcomer issue for someone else to sort them ;-)